docs/guides/jupyter.rst

```diff
@@ -0,0 +1,377 @@
+How to use Jupyter notebooks in Sphinx
+======================================
+
+`Jupyter`_ notebooks are a popular tool to describe computational narratives
+that mix code, prose, images, interactive components, and more.
+Embedding them in your Sphinx project allows using these rich documents as documentation,
+which can provide a great experience for tutorials, examples, and other types of technical content.
+There are a few extensions that allow integrating Jupyter and Sphinx,
+and this document will explain how to achieve some of the most commonly requested features.
+
+.. _Jupyter: https://jupyter.org/
+
+By default, `Sphinx only supports reStructuredText source files for
+documentation <https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-source_suffix>`_.
+Luckily, there are a few extensions to also allow Jupyter notebooks as source files.
+
+.. _nb-background:
+
+Background on the existing relevant extensions
+----------------------------------------------
+
+There are two main extensions that add support Jupyter notebooks as source files in Sphinx:
+nbsphinx_ and MyST-NB_. Although they have similar intent and basic functionality,
+there are some differences:
+
+- nsphinx uses `pandoc <https://pandoc.org/>`_ to convert the Markdown from Jupyter notebooks
+  to reStructuredText and then to docutils AST,
+  whereas MyST-NB uses `MyST-Parser`_ to directly convert the Markdown text to docutils AST.
+  Both Markdown flavors are mostly equal, but they have some differences.
+- Both nbsphinx and MyST-NB support canonical ``.ipynb`` notebooks.
+  nbsphinx can process any format supported by `jupytext`_,
+  whereas MyST-NB focuses on MyST Markdown Notebooks
```

Comment:
myst-nb can also support any format: https://myst-nb.readthedocs.io/en/latest/examples/custom-formats.html

Reply:
Thanks for the clarification! That was a misunderstanding on my side, will fix this (applies to your other comments below)